ui/app/services/version.js

```diff
@@ -22,7 +24,14 @@ export default class VersionService extends Service {
     return !this.isEnterprise;
   }
 
```

Comment:
Normally, I'd avoid moving methods unecessarily. However, it makes it a lot easier to read this file if all the license getters are grouped together.